docs/kusion/3-concepts/5-appconfiguration.md

```diff
@@ -5,56 +5,14 @@ sidebar_label: AppConfiguration
 
 # AppConfiguration
 
-## Abstract
+As a modern cloud-native application delivery toolchain, declarative intent-based actuation is the central idea of Kusion, `AppConfiguration` model plays the role of describing the intent, which provides a simpler path for on-boarding developers to the platform without leaking low level details in runtime infrastructure and allows developers to fully focus on the application delivery itself.
 
-Kusion consumes the declarative configuration that describes the application, and delivers intent to the target runtime including Kubernetes, clouds, or on-prem infrastructure. In order to do that, we need a declarative model that can best describe the intent for a given application.
-
-This design document elaborates on the core design considerations and a high-level architecture of the next-generation declarative application configuration model. The AppConfiguration model is expected to continuously iterate over time, with the purpose to better encapsulate the application configuration needs declaratively in its full lifecycle and reduce cognitive complexity as much as possible for the developers.
-
-AppConfiguration consists of multiple elements that each represents a significant portion in the application lifecycle, either the application workload itself, its dependencies, relevant deployment workflows or operational expectations.
-
-## Motivation
-
-In AntGroup, we have heavily invested in the efforts to enable application delivery on a massive scale. What we have observed in the past few years is a trend of continuous evolution of infrastructure complexity over time, as a result of the increasing business needs.
-
-We are motivated to find a new paradigm that highlights collaborations between different parts of the software organizations and enables self-service ability as much as possible to get to a mature level of standardization and efficiency in application delivery.
-
-The centerpiece of this paradigm is a consistent, comprehensive and declarative application model that captures the application needs in an intuitive and self-explanatory way.
-
-This paradigm will also require a core workflow that KusionStack advocates, an effort of retrofitting that workflow based on different organizational needs, a golden path that represents industry best practices, and a shift in organizational culture. We won't go into those details in this document.
-
-## Design
```

Comment:
Why are we removing the design considerations and core principles? Are we planning on moving this somewhere?

Reply:
yes, we plan moving this to design doc, also post in `https://blog.kusionstack.io/`.